pages/common/aws-kinesis.md

```diff
@@ -0,0 +1,28 @@
+# aws kinesis
+
+> Official AWS CLI for Amazon Kinesis streaming data services.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/kinesis/index.html#cli-aws-kinesis>.
+
+- Show all streams in the account:
+
+`aws kinesis list-streams`
+
+- Write one record to a Kinesis stream:
+
+`aws kinesis put-record --stream-name {{streamname}} --partition-key {{partition_key}} --data {{base64_encoded_message}}`
+
+- Write a record to a Kinesis stream with inline base64 encoding:
+
+`aws kinesis put-record --stream-name {{streamname}} --partition-key {{partition_key}} --data "$( echo "{{my raw message}}" | base64 )"`
+
+- List the shards available on a stream:
+
+`aws kinesis list-shards --stream-name {{streamname}}`
+
+- Get a shard iterator for reading from the oldest message in a stream's shard:
+
+`aws kinesis get-shard-iterator --shard-iterator-type TRIM_HORIZON --stream-name {{streamname}} --shard-id {{shard_id}}`
+
+- Read records from a shard, using a shard iterator (e.g., retrieved with get-shard-iterator):
```

Comment:
IMO, there is no need to mention the e.g. part. The earlier example clearly mentions "Get a shard iterator". So it's sufficient to just say "using a given shard iterator"

Reply:
Makes sense, removing in 595a3c6a.